python3-get-framework-k8s-api-version.spec

```diff
@@ -17,17 +17,20 @@
 
 
 %define upstream_name get-framework-k8s-api-version
+%define is_aws "%(if [ -e /sys/hypervisor/uuid ]; then grep -r ^ec2 /sys/hypervisor/uuid; else echo 0; fi)"
```

Comment:
Please ad a comment w.r.t. the limitation for this definition. For example CAP images will get built in IBS and as such this macro will not ever trigger the `Requires: python3-boto3` during image build.

This is a double edged sword, at the very least we need a comment in the spec file.